index.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html lang="en-us">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="stylesheet" href="./assets/css/style.css" />
+    <title>Horiseon Homepage</title>
+  </head>
+
+  <body>
+    <!-- this section of code refers to header of page for title and navigation -->
+    <header>
+      <h1>Hori<span class="seo">seo</span>n</h1>
+      <nav>
+        <ul>
+          <li>
+            <a href="#search-engine-optimization">Search Engine Optimization</a>
+          </li>
+          <li>
+            <a href="#online-reputation-management">Online Reputation Management</a>
+          </li>
+          <li>
+            <a href="#social-media-marketing">Social Media Marketing</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <!-- hero class defines background image -->
+    <div class="hero"></div>
+    <!-- this 'main' block contains 3 sections -->
+    <main>
+      <section id="search-engine-optimization">
+        <img
+          src="./assets/images/search-engine-optimization.jpg"
+          class="float-left"
+          alt="Plan view of desk, laptop, coffee, stationary."
+        />
+        <h2>Search Engine Optimization</h2>
+        <p>
+          The dominance of mobile internet use means that users are searching
+          for the right business as they travel, shop, or sit on their couch at
+          home. Search Engine Optimization (SEO) allows you to increase your
+          visibility and find the right customers for your business.
+        </p>
+      </section>
+      <section id="online-reputation-management">
+        <img
+          src="./assets/images/online-reputation-management.jpg"
+          class="float-right"
+          alt="Laptop showing a reputation graph increasing."
+        />
+        <h2>Online Reputation Management</h2>
+        <p>
+          The web is full of opinions, and some of these can be negative. Social
+          media allows anyone with an internet connection to say whatever they
+          want about your business. Online Reputation Management gives you the
+          control over what potential customers see when they search for your
+          business.
+        </p>
+      </section>
+      <section id="social-media-marketing">
+        <img
+          src="./assets/images/social-media-marketing.jpg"
+          class="float-left"
+          alt="Birds eye view of desk with people collaborating."
+        />
+        <h2>Social Media Marketing</h2>
+        <p>
+          Social media continues to have a sizable influence on buying habits.
+          Social media marketing helps you determine which platforms are suited
+          to your brand, using analytics to find the right markets and increase
+          your lead generation.
+        </p>
+      </section>
+    </main>
+    <!-- this 'aside' refers to our sidebar -->
+    <aside>
+      <div>
```

Comment:
You could also use a section tag instead of a `div` since this can be considered as 3 sections within the `aside` tag. But `div` is also fine in this case. You can read more about `section` vs `div` [here](https://medium.com/design-code-repository/html-elements-section-vs-div-vs-article-a8c34e6548cf)